src/main/java/com/sahokia/kos/hibernatehomework/model/Customer.java

```diff
@@ -0,0 +1,60 @@
+package com.sahokia.kos.hibernatehomework.model;
+import jakarta.persistence.*;
+
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "customer")
+public class Customer {
+
+    public Customer() {}
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "creation_date")
+    private LocalDate creationDate;
```

Comment:
Better to store Instant(full date with time and if we need only date we could filter it later while mapping), use @CreationTimestamp